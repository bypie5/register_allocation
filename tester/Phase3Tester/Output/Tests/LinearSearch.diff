--- SelfTestCases/LinearSearch.out	2020-03-05 22:31:29.000000000 -0800
+++ ./Output/Tests/LinearSearch.out	2020-03-09 14:31:07.000000000 -0700
@@ -1,15 +1,5 @@
-10
-11
-12
-13
-14
-15
-16
-17
-18
-9999
-0
-1
-1
-0
-55
+--- Error (ProgramFault) ---
+require function address, found integer10
+  Main() line 20
+  LS.Start() line 39
+  LS.Init() line 189
