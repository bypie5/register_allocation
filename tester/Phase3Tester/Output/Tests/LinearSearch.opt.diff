--- SelfTestCases/LinearSearch.opt.out	2020-03-05 22:31:29.000000000 -0800
+++ ./Output/Tests/LinearSearch.opt.out	2020-03-08 18:43:31.000000000 -0700
@@ -1,15 +1,3 @@
-10
-11
-12
-13
-14
-15
-16
-17
-18
-9999
-0
-1
-1
-0
-55
+--- Error (ProgramFault) ---
+require function address, found data address<Heap.0>+0
+  Main() line 11
