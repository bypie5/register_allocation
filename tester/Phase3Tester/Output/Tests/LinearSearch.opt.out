--- Error (ProgramFault) ---
require function address, found data address<Heap.0>+0
  Main() line 11
